doc-General_Configuration/topics/Configuration.adoc

```diff
@@ -2052,6 +2052,52 @@ To modify a schedule:
 . Make the required changes.
 . Click *Save*.
 
+
+[[import-export-schedule]]
+===== Importing and Exporting Schedules
+
+{product-title_short} provides the ability to import and export schedules using the `import_export_schedules.rb` script:
+
+----
+./tools/import_export_schedules.rb -h
```

Comment:
**Suggestion:** 
@suyogsainkar From which location need to execute it, can you please add exact path or create alias for it. ?

Reply:
> **Suggestion:**
> @suyogsainkar From which location need to execute it, can you please add exact path or create alias for it. ?
@sbulage I haven't tested this myself. Can you please share info about the location/exact path to execute it? If not, I'll check with engineering. Thanks.

Reply:
Okay

Reply:
@suyogsainkar This is the full path where `import_export_schedules.rb` file resides.
```
cd /var/www/miq/vmdb/tools
import_export_schedules.rb -h
```

Reply:
Thanks Satyajit!